generate-clients.sh

```diff
@@ -48,10 +49,14 @@ function generate_client() {
       --git-host "github.com" --git-user-id RedHatInsights --git-repo-id patchman-clients \
       --type-mappings DateTime=string \
       -o /local/$NAME
+
+    if [ "$NAME" == "inventory" ] ; then
+        sed -i 's/uNKNOWNBASETYPE/createCheckIn/g; s/UNKNOWN_BASE_TYPE/CreateCheckIn/g;' $NAME/api_hosts.go
```

Comment:
Have you an idea how does it happen?

Reply:
I see.

Reply:
Just for the record - new stable openapigenerator v5.0.1 throws the same error.

Reply:
Ok. Btw. if there is new version of generator available we could upgrade it. But I would do that as a separate step. So we will have both versions in master - I mean the one generated by old version and the second generated by the new one. It will be also more clear what changes were done in v5.x.x.